libkineto/src/RoctracerActivityApi.h

```diff
@@ -135,41 +55,25 @@ class RoctracerActivityApi {
     const std::set<ActivityType>& selected_activities);
   void clearActivities();
 
-  int processActivities(ActivityLogger& logger);
+  int processActivities(ActivityLogger& logger
+                        , std::function<const ITraceActivity*(int32_t)> linkedActivity);
```

Comment:
nit, comma should be in the previous line

Reply:
ty